### ISSUE TITLE
Accessing Presets/Samples in .zip folder

### ISSUE THREAD

**Issue by GingkathFox** [LMMS/lmms#4474]
This is good for people with small HDs.
Is it possible to access files within a .zip file (or any other archive format)?

**Comment by tresf** [comment]
> Is it possible to access files within a .zip file (or any other archive format)?

No.  I've never used a system -- ever -- that allows a zip file contents to be accessed as a standard file system path.  Sounds a bit like what [this user was trying to do](https://github.com/LMMS/lmms/issues/2569), but more pragmatic. :)

But seeing into the future would mark this as a duplicate of #3981.  Project should be exportable into zip files.  If you want to [compress a volume, just do it.  No one is stopping you](https://www.windowscentral.com/how-use-ntfs-compression-windows-10).

**Comment by tresf** [comment]
@PhysSong I don't agree that this is `ux` or `enhancement`.  I believe this is `needs clarification`.  What the OP is describing is not standard desktop behavior in any app I've ever seen.
